scrapers/MissaX.py

```diff
@@ -0,0 +1,132 @@
+#this scraper scrapes title and uses it to search the site and grab a cover from the search results, among other things
+import base64
+import datetime
+import json
+import re
+import sys
+import urllib.parse
+# extra modules below need to be installed
+try:
+    import py_common.log as log
+except ModuleNotFoundError:
+    print("You need to download the folder 'py_common' from the community repo! (CommunityScrapers/tree/master/scrapers/py_common)", file=sys.stderr)
+    sys.exit()
+
+try:
+    import cloudscraper
+except ModuleNotFoundError:
+    log.error("You need to install the cloudscraper module. (https://pypi.org/project/cloudscraper/)")
+    log.error("If you have pip (normally installed with python), run this command in a terminal (cmd): pip install cloudscraper")
+    sys.exit()
+    
+try:
+    from lxml import html, etree
+except ModuleNotFoundError:
+    log.error("You need to install the lxml module. (https://lxml.de/installation.html#installation)")
+    log.error("If you have pip (normally installed with python), run this command in a terminal (cmd): pip install lxml")
+    sys.exit()
+
+STUDIO_MAP = {
+    'https://missax.com/tour/': 'MissaX',
+    'https://allherluv.com/tour/': 'All Her Luv',
+}
+
+proxy_list = {
+    }
+timeout = 10
+
+def scraped_content(scraper, url):
+    try:
+        scraped = scraper.get(url, timeout=timeout, proxies=proxy_list)
+    except:
+        log.error("scrape error")
+    if scraped.status_code >= 400:
+        log.error('HTTP Error: %s' % scraped.status_code)
+    return scraped.content
+
+def scrape_scene_page(url): #scrape the main url
+    tree = scraped_content(scraper, url) #get page content
+    tree = html.fromstring(tree) #parse html
+    title = tree.xpath('//p[@class="raiting-section__title"]/text()')[0].strip() #title scrape
+    log.trace(f'Title:{title}')
```

Comment:
use `log.debug` instead of log.trace (for the rest of the prints as well)
imho for debuging scrapers log.trace prints too much info